src/Cache/MongoStore.php

```diff
@@ -181,12 +182,9 @@ public function get($key): mixed
     #[Override]
     public function increment($key, $value = 1): int|float|false
     {
-        $this->forgetIfExpired($key);
-
         $result = $this->collection->findOneAndUpdate(
             [
                 '_id' => $this->prefix . $key,
-                'expiration' => ['$gte' => $this->currentTime()],
```

Comment:
By removing this criteria and the preceding call to `forgetIfExpired()`, you're potentially incrementing an expired document. Is that a concern?

Since the TTL index is optional, I wouldn't have expected this to change.

Reply:
I've removed the previous `forgetIfExpired` in order to avoid useless command. A nominal usage must be a single command.

Even if the expired item is incremented here, it is deleted just after.